web/profiles/joinup/config/install/og.og_role.rdf_entity-solution-member.yml

```diff
@@ -0,0 +1,11 @@
+langcode: en
+status: true
+dependencies: {  }
+id: rdf_entity-solution-member
+label: Member
+weight: 1
+group_type: rdf_entity
+group_bundle: solution
+group_id: null
+permissions: { }
+role_type: immutable
```

Comment:
This has changed, this `role_type` is no longer called `immutable`, but `required`. Weird to see it appear like this. I just double checked and this change happened around 2 months ago in OG. Did you work maybe from an installation that was done from an older version of OG?

Or perhaps we're just carrying this around for a long time now. OG doesn't have update hooks yet so the data is never updated when something changes.